sdcm/nemesis.py

```diff
@@ -1828,6 +1828,10 @@ def _prepare_test_table(self, ks='keyspace1', table=None):
                 stress_cmd=stress_cmd, keyspace_name=ks, stop_test_on_failure=False, round_robin=True)
             cs_thread.verify_results()
 
+    @scylla_versions(("5.2.rc0", None), ("2023.1.rc0", None))
```

Comment:
Need also implement the method for older versions. For the moment older scylla versions still will get failures -> `MethodVersionNotFound` exception will be raised.

Reply:
fixed

Reply:
doesn't look so. @fgelcer are you sure it works?

Reply:
@fgelcer the latest change made at https://github.com/scylladb/scylla-cluster-tests/commit/854e130e798780e81699e3d77e1c989ee9b7c0fd is totally wrong.
The code snippet must have been applied `as is`.

Reply:
@fgelcer also, need to add unit test coverage for both syntax variants. With proper unit tests you would see that current code is broken.